packages/core/test/NodeExecuteFunctions.test.ts

```diff
@@ -543,6 +545,48 @@ describe('NodeExecuteFunctions', () => {
 	});
 
 	describe('parseRequestObject', () => {
+		test('should handle basic request options', async () => {
+			const axiosOptions = await parseRequestObject({
+				url: 'https://example.com',
+				method: 'POST',
+				headers: { 'content-type': 'application/json' },
+				body: { key: 'value' },
+			});
+
+			expect(axiosOptions).toEqual(
+				expect.objectContaining({
+					url: 'https://example.com',
+					method: 'POST',
+					headers: { accept: '*/*', 'content-type': 'application/json' },
+					data: { key: 'value' },
+					maxRedirects: 0,
+				}),
+			);
+		});
+
+		test('should set correct headers for FormData', async () => {
+			const formData = new FormData();
+			formData.append('key', 'value');
+
+			const axiosOptions = await parseRequestObject({
+				url: 'https://example.com',
+				formData,
+				headers: {
+					'content-type': 'multipart/form-data',
+				},
+			});
+
+			expect(axiosOptions.headers).toMatchObject({
+				accept: '*/*',
+				'content-length': 163,
+				'content-type': expect.stringMatching(
+					/^multipart\/form-data; boundary=--------------------------[0-9]{24}$/,
+				),
```

Comment:
We could also choose not to be so strict in case `FormData` changes how it implements the boundary. We mostly care about this being `multipart/form-data` or?

```ts
  expect(axiosOptions.headers['content-type']).toMatch(/^multipart\/form-data; boundary=/);
```